assets/cubyz/biomes/taiga.json

```diff
@@ -1,29 +1,29 @@
-{
-	"properties" : [
-		"cold"
-	],
-	"minHeight" : 22,
-	"maxHeight" : 40,
-	
-	"roughness" : 10,
-	"hills" : 10,
-
-	"validPlayerSpawn" : true,
-	
-	"ground_structure" : [
-		"0 to 1 cubyz:grass",
-		"2 to 3 cubyz:soil"
-	],
-	"structures" : [
-		{
-			"id" : "cubyz:simple_tree",
-			"leaves" : "cubyz:oak_leaves",
-			"log" : "cubyz:oak_log",
-			"top" : "cubyz:oak_top",
-			"chance" : 0.05,
-			"type" : "pyramid",
-			"height" : 8,
-			"height_variation" : 5
-		}
-	]
-}
+{
+	"properties" : [
+		"cold"
+	],
+	"minHeight" : 22,
+	"maxHeight" : 40,
+	
+	"roughness" : 10,
+	"hills" : 10,
+
+	"validPlayerSpawn" : true,
+	
+	"ground_structure" : [
+		"0 to 1 cubyz:grass",
+		"2 to 3 cubyz:soil"
+	],
+	"structures" : [
+		{
+			"id" : "cubyz:simple_tree",
+			"leaves" : "cubyz:pine_needles",
+			"log" : "cubyz:pine_log",
+			"top" : "cubyz:pine_top",
+			"chance" : 0.025,
+			"type" : "pyramid",
+			"height" : 8,
+			"height_variation" : 5
+		}
+	]
+}

```

Comment:
It looks like you have the incorrect line endings again.
Sadly my automated script wasn't detecting this correctly.

Reply:
Is there a way to run the check again, now that you've fixed it? Or does it do that when the PR is merged?

Reply:
It automatically runs it every time you make a new commit.
But you need the new script in your branch, so you'd have to update your branch to the latest commit from `master`.